conda/environment.yml

```diff
@@ -8,14 +8,16 @@ channels:
   - conda-forge
 dependencies:
   - ipyleaflet
-  - jupyter-book<2
+  - jupyter-book
   - jupyter-resource-usage
   - jupyterhub-singleuser
   - jupyterlab
   - jupytext
   - pip
   - python
+  - sphinx
   - sphinxcontrib-bibtex
+  - yaml
```

Comment:
jupyter-book brings these in already (`mamba repoquery whoneeds sphinx`) so I recommend just keeping this file unchanged